src/content/docs/browser/new-relic-browser/configuration/filter-ajax-request-events.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Filter AjaxRequest events'
+metaDescription: "For New Relic browser monitoring: use the app settings page to filter which network calls are recorded as AjaxRequest events."
+---
+In agent version v1211 or higher, all network requests made by a page are recorded as AjaxRequest events. Included with this change is a new deny list configuration option available in the application settings page to filter which requests record events.
+Regardless of this filter, all network requests are captured as metrics and available in the AJAX Page.
+
+To update the deny list of URLs your application will filter from creating events, go to the app settings page:
+1. Go to [one.newrelic.com](http://one.newrelic.com), and click **Browser**. 
+2. Select an app. 
+3. On the left navigation, click **App settings**.
+4. Under **Ajax Request Deny List**, add the filters you would like to apply to your app.
+
+   <Callout variant="important">
```

Comment:
This is a very large callout in the middle of the procedure. Usually we keep callouts short, maybe a sentence. Since this info is important for how they are going to set up their deny list, would it be more helpful to pull the info out of the callout and, instead, present this info before the procedure begins? That way you are giving them guidelines about how to approach their deny list config before they go into the UI.